core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -451,6 +451,8 @@ sealed abstract class IO[+E, +A] extends Serializable { self =>
   /**
    * Repeats this action with the specified schedule until the schedule
    * completes, or until the first failure.
+   * Repeat are done in addition of the first execution so that
+   * `io.repeat(Schedule.once)` means "execute io and in case of success repeat io one time".
```

Comment:
You mean "repeat io once" in place of `io.repeat(Schedule.once)`?
If so, I would prefer not to do that change for the following reason: I wanted to write down the code, because the semantic of repeat/retry/execute is not very clear and shared among people with different backgrounds and habits (or foreign lang users like me, where the traduction may loose some important shades). So I wanted to let people equate an actual piece of code with the corresponding wordings. 
Of course, I'm happy to change that if the reason doesn't hold (or if I didn't understand what you meant)

Reply:
I think he meant just replacing the end of your sentence `one time` -> `once`

Reply:
ah :) perfect then!